content/docs/services/cloud-gov-identity-provider.md

```diff
@@ -4,7 +4,7 @@ menu:
     parent: services
 title: cloud.gov identity provider
 name: "cloud-gov-identity-provider"
-description: "Manage client credentials for authenticating cloud.gov users in your app"
+description: "Authenticate cloud.gov users in your app"
```

Comment:
Hmm, although given an oauth token, potentially you could do much more than just use cloud.gov as a login. You could access any cloud.gov resources on behalf of the user. Do we have examples of what users are building with this service?